mlrun/runtimes/sparkjob.py

```diff
@@ -230,6 +234,19 @@ def _run(self, runobj: RunObject, execution: MLClientCtx):
         update_in(job, "spec.driver.volumeMounts", self.spec.volume_mounts)
         update_in(job, "spec.executor.volumeMounts", self.spec.volume_mounts)
         update_in(job, "spec.deps", self.spec.deps)
+
+        if self.spec.spark_conf:
+            job["spec"]["sparkConf"] = {}
+            for k, v in self.spec.spark_conf.items():
+                job["spec"]["sparkConf"]['"{}"'.format(k)] = '"{}"'.format(v)
+        #                update_in(job, 'spec.sparkConf."' + k + '"', '"' + v + '"')
```

Comment:
* Remove the commented code
* I suggest to change to:
```suggestion
                job["spec"]["sparkConf"][f'"{k}"'] = f'"{v}"'
```

Reply:
Also, what values can come inside this `spark_conf` and `hadoop_conf` ? 
There's a chance they'll include quotes (`"`) themselves ? if so, you'll need to use `shlex.quote` instead of wrapping with quotes manually

Reply:
I don't think that " are an option (according to the examples I see in the spark operator documentation).
I think that this is mainly to avoid issues with : or .